tests/test_plotting.py

```diff
@@ -60,581 +60,489 @@ def test_kwarg_warning(DummyStellarator):
-        expand_out=False,
-    )
-    np.testing.assert_allclose(
-        plot_data["<" + name + ">_fsa"], desired, equal_nan=False
-    )
+    @pytest.mark.unit
+    @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
+    def test_1d_iota_radial(self):
+        """Test plotting 1d rotational transform."""
+        eq = get("DSHAPE_current")
+        fig, ax, data = plot_1d(eq, "iota", figsize=(4, 4), return_data=True)
+        assert "rho" in data.keys()
+        assert "iota" in data.keys()
+        return fig
 
-    name = "|B|"
-    assert (
-        "<" + name + ">" in data_index["desc.equilibrium.equilibrium.Equilibrium"]
-    ), "Test with a different quantity."
-    _, _, plot_data = plot_fsa(
-        eq=eq,
-        name=name,
-        rho=rho,
-        M=eq.M_grid,
-        N=eq.N_grid,
-        with_sqrt_g=False,  # Test that does not compute data_index["<|B|>"]
-        return_data=True,
-    )
-    data = eq.compute(names=name, grid=grid)
-    desired = surface_averages(grid=grid, q=data[name], expand_out=False)
-    np.testing.assert_allclose(
-        plot_data["<" + name + ">_fsa"], desired, equal_nan=False
-    )
+    @pytest.mark.unit
+    @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
+    def test_1d_logpsi(self):
+        """Test plotting 1d flux function with log scale."""
+        eq = get("DSHAPE_current")
+        fig, ax, data = plot_1d(eq, "psi", log=True, figsize=(4, 4), return_data=True)
+        ax.set_ylim([1e-5, 1e0])
+        assert "rho" in data.keys()
+        assert "psi" in data.keys()
+        return fig
 
+    @pytest.mark.unit
+    @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
+    def test_plot_1d_curve(self):
+        """Test plot_1d function for Curve objects."""
+        curve = FourierXYZCurve([0, 10, 1])
+        fig, ax = plot_1d(curve, "curvature")
+        return fig
 
-@pytest.mark.unit
-@pytest.mark.solve
-@pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
-def test_fsa_I(DSHAPE_current):
-    """Test plotting of flux surface average toroidal current."""
-    eq = load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
-    fig, ax, data = plot_fsa(eq, "B_theta", with_sqrt_g=False, return_data=True)
-    assert "rho" in data.keys()
-    assert "<B_theta>_fsa" in data.keys()
-    assert "normalization" in data.keys()
-    assert data["normalization"] == 1
+    @pytest.mark.unit
+    @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
+    def test_plot_1d_surface(self):
+        """Test plot_1d function for Surface objects."""
+        surf = FourierRZToroidalSurface()
+        fig, ax = plot_1d(surf, "curvature_H_rho", grid=LinearGrid(M=50))
+        return fig
 
-    return fig
 
+class TestPlot2D:
+    """Tests for plot_2d."""
 
-@pytest.mark.unit
-@pytest.mark.solve
-@pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
-def test_fsa_G(DSHAPE_current):
```

Comment:
redundant with `test_fsa_I`

Reply:
If anyone else was initially uncomfortable with removing these tests, I'm okay with it since I think it's redundant with the two tests (`test_fsa_I` and `test_compute_everything`).